test/functional/readconcern_tests.js

```diff
@@ -94,37 +96,42 @@ describe('ReadConcern', function() {
       test: function(done) {
         // Get a new instance
         const configuration = this.configuration;
-        client = !test.urlOptions
-          ? configuration.newClient({ w: 1 }, { poolSize: 1, readConcern: { level: test.level } })
-          : null;
+        let options;
 
         if (test.urlReadConcernLevel || test.urlOptions) {
           url = configuration.url();
           if (!test.urlOptions) {
```

Comment:
This should be `test.urlOptions != null` because `!test.urlOptions` could return true if `test.urlOptions` is falsy.